fix_includes.py

```diff
@@ -2169,6 +2173,13 @@ def main(argv):
   parser.add_option('--nosafe_headers', action='store_false',
                     dest='safe_headers')
 
+  parser.add_option('--reorder', default=False, action='store_true', dest='reorder',
+                    help=('Re-order lines relative to other similar lines '
+                          '(eg, headers relative to other headers)'))
+
+  parser.add_option('--no-reorder', default=False, action='store_false', dest='reorder',
```

Comment:
And this is why you don't need a default: Using `--no-reorder` will store `false` into `reorder`, not using the flag will store the default `false` into `reorder` making this whole option having no effect.

I'd suggest you go for exactly 1 of those 2